test/e2e/proxy_test.go

```diff
@@ -88,13 +98,27 @@ func TestProxyFlow(t *testing.T) {
 						"planet": "venus",
 					},
 				}
+				// Start a new websocket watcher which watches for config maps in the user's namespace
+				w := newWsWatcher(t, user, hostAwait.APIProxyURL)
+				closeConnection := w.Start()
+				defer closeConnection()
 
 				// when
 				proxyCl := hostAwait.CreateAPIProxyClient(user.token)
 				err := proxyCl.Create(context.TODO(), expectedCM)
 				require.NoError(t, err)
 
 				// then
+				// wait for the websocket watcher which uses the proxy to receive the config map
+				found, err := w.WaitForConfigMap(
```

Comment:
what about testing that multiple resources can be received with the same websocket connection?